pulumi/grapl/__main__.py

```diff
@@ -479,6 +502,9 @@ def main() -> None:
         vars=grapl_core_vars,
         opts=pulumi.ResourceOptions(
             provider=nomad_provider,
+            depends_on=[
```

Comment:
Does the order of whether we deploy grapl-core vs the graph-db services actually matter here or is this just to make sure that we have a consistent deployment order?

Reply:
Doesn't really matter. And could potentially slowdown the overall deploy. So I'm fine ripping that out